x-pack/plugins/maps/public/classes/util/assign_feature_ids.ts

```diff
@@ -43,6 +46,7 @@ export function assignFeatureIds(featureCollection: FeatureCollection): FeatureC
       properties: {
         // preserve feature id provided by source so features can be referenced across fetches
         [FEATURE_ID_PROPERTY_NAME]: feature.id == null ? numericId : feature.id,
+        [KBN_TOO_MANY_FEATURES_PROPERTY]: false,
```

Comment:
This seems weird that you have to add `KBN_TOO_MANY_FEATURES_PROPERTY` to every property. How about removing and then changing `const TOO_MANY_FEATURES_FILTER = ['all', ['!=', ['get', KBN_TOO_MANY_FEATURES_PROPERTY], true]];`?